utils/readPkgUp.js

```diff
@@ -35,10 +36,40 @@ function stripBOM(str) {
  * THE SOFTWARE.
  */
 exports.default = function readPkgUp(opts) {
-  const fp = pkgUp(opts);
+  if (opts.multi) {
+    const allPackages = [];
+    let searchPath = opts.cwd;
+    const root = path.parse(searchPath).root;
+    while (searchPath) {
+      const pkg = readPkgUp(Object.assign({}, opts, {
+        multi: false,
+        cwd: searchPath,
+        requireExact: opts.requireExact && allPackages.length === 0,
+      }));
```

Comment:
it would be ideal not to complicate this largely inlined dependency with a new option.